test/ssl-tests/02-protocol-version.cnf.in

```diff
@@ -16,4 +16,9 @@ use warnings;
 
 use protocol_version;
 
-our @tests = generate_version_tests("TLS");
+our @tests;
+if (FIPS_MODE) {
+    @tests = generate_version_tests("TLS", 1);
+} else {
+    @tests = generate_version_tests("TLS", 0);
+}
```

Comment:
Why not this?

``` perl
our @tests = generate_version_tests("TLS", FIPS_MODE);
```

... or well, with the change request i just made:

``` perl
our @tests = generate_version_tests("TLS", $fips_mode);
```

Reply:
Done